### PR TITLE
Journal related tags for google scholar

### PR DESCRIPTION
**Pull request by anusharanganathan**
PR for https://github.com/antleaf/nims-mdr-development/issues/315
It sets journal meta tags for publications

### CODE REVIEW

**Comment by nabeta** [comment]
@anusharanganathan Could you add those tags to Dataset in addition to Publication?

**Comment by anusharanganathan** [comment]
@nabeta these tags should now be visible for all works, if the journal data exists

**Comment by anusharanganathan** [comment]
This is a PR for https://github.com/antleaf/nims-mdr-development/issues/315

File: hyrax/config/locales/en.yml [hunk 1]
```diff
@@ -86,6 +86,7 @@ en:
         first_published_url: "https://doi.org/10.xxx/xxx"
         managing_organization: NIMS
         managing_organization_ordered: NIMS
+        supervisor_approval: approved
```
Comment (asahiko) [inline hyrax/config/locales/en.yml:89]: To resolve merge conflict

**Review by nabeta — APPROVED** [review]

**Pull request merged**